openstudiocore/ruby/openstudio/sketchup_plugin/lib/dialogs/SpaceAttributesInterface.rb

```diff
@@ -174,16 +177,27 @@ module OpenStudio
           else
             assigned_const_set = drawing_interface.model_object.setString(3,selected_const_set.to_s)
           end
-
-          # set space's parent zone values
-          # (I could probalby make array of unique values and loop outside of main loop so I don't reset the string over and over for spaces that share zones)
+          
+          # get thermostat object from input name
+          # make thermostat list
+          thermostat_objects = model.getThermostats
+          thermostat_objects.each do |object|
+            if object.name.to_s == thermostat
+              thermostat = object
```

Comment:
@DavidGoldwasser this is a style comment, here you are reusing the thermostat variable which used to be a string to now be the actual model object.  I would normally use a second variable initialized to nil to hold the model object, that way i could check if i found the object or not.  I guess here you know you will find the object because the list of names is populated right before this runs, but in general you can't always know that you are going to find the object you are looking for.